### PR TITLE
add readme for atomic dp

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/molecules/molecules-readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Patterns to follow while creating molecules
+- Composed of other molecules or atoms.
+- Use CSS to style .
+- Dumb components avoid using logic at this level.
+- May use slots.
```
Comment (arrogant-ant) [inline src/components/molecules/molecules-readme.md:5]: what do you mean by slots here

File: src/components/molecules/molecules-readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Patterns to follow while creating molecules
+- Composed of other molecules or atoms.
+- Use CSS to style .
+- Dumb components avoid using logic at this level.
```
Comment (arrogant-ant) [inline src/components/molecules/molecules-readme.md:4]: avoid using business logic
may be we can use `generic validations`

File: dist/bangalore/parking-near-banashankari/index.html [hunk 3]
```diff
@@ -26,4 +26,4 @@
         dataLayer.push(arguments);
       }
       gtag("js", new Date());
-      asyncgtag("config", "G-779VHE86MJ");</script><script src="/js/chunk-vendors.d45e0b3e.js"></script><script src="/js/app.e0150a1e.js"></script></body></html>
\ No newline at end of file
+      asyncgtag("config", "G-779VHE86MJ");</script><script src="/js/chunk-vendors.d45e0b3e.js"></script><script src="/js/app.e0150a1e.js"></script><iframe style="display: none;"></iframe></body></html>
```
Comment (arrogant-ant) [inline dist/bangalore/parking-near-banashankari/index.html:29]: why dist has changed?

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/molecules/molecules-readme.md (hunk 1 above)
Comment (alias01) [reply src/components/molecules/molecules-readme.md:5]: Ways to pass data to child components.

File: src/components/molecules/molecules-readme.md (hunk 2 above)
Comment (alias01) [reply src/components/molecules/molecules-readme.md:4]: yes validation is valid...will include

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**